tests/unittests/Foundation/NSStringInternalTests.mm

```diff
@@ -14,6 +14,8 @@
 //
 //******************************************************************************
 
+#ifdef _WIN32
```

Comment:
Feels like we need a comment to explain the differences when we have them (especially when whole tests / files are commented out). I'm a little wary of how many of these we are sprinkling around.